terraform/helm/validator-lite/files/configs/validator.yaml

```diff
@@ -0,0 +1,105 @@
+base:
+  waypoint:
+    from_storage:
+      type: "vault"
+      server: "{{ .Values.vault.server.address }}"
+      {{- if .Values.vault.server.ca_cert }}
+      ca_certificate: "{{ .Values.vault.server.ca_cert }}"
+      {{- end }}
+      namespace: "{{ .Values.vault.namespace }}"
+      token:
+        from_disk: "/opt/vault/token"
+
+consensus:
+  safety_rules:
+    service:
+      type: "local"
+    backend:
+      type: "vault"
+      server: "{{ .Values.vault.server.address }}"
+      {{- if .Values.vault.server.ca_cert }}
+      ca_certificate: "{{ .Values.vault.server.ca_cert }}"
+      {{- end }}
+      namespace: "{{ .Values.vault.namespace }}"
+      token:
+        from_disk: "/opt/vault/token"
+  sync_only: {{ .Values.validator.config.sync_only | default false}}
+  {{- if .Values.validator.config.mempool_poll_count }}
+  mempool_poll_count: {{ .Values.validator.config.mempool_poll_count }}
+  {{- end }}
+  {{- if .Values.validator.config.round_initial_timeout_ms }}
+  round_initial_timeout_ms: {{ .Values.validator.config.round_initial_timeout_ms }}
+  {{- end }}
+
+storage:
+  storage_pruner_config:
+    default_prune_window: {{ int .Values.validator.config.default_prune_window }}
+    state_store_prune_window: {{ int .Values.validator.config.state_store_prune_window }}
+    max_version_to_prune_per_batch: {{ int .Values.validator.config.max_version_to_prune_per_batch }}
+
+execution:
+  {{- if eq .Values.validator.config.provide_genesis true }}
+  genesis_file_location: "../genesis/genesis.blob"
+  {{- else }}
+  genesis_file_location: ""
+  {{- end }}
+
+validator_network:
+  discovery_method: "onchain"
+  mutual_authentication: true
+  identity:
+    type: "from_storage"
+    key_name: "validator_network"
+    peer_id_name: "owner_account"
+    backend:
+      type: "vault"
+      server: "{{ .Values.vault.server.address }}"
+      {{- if .Values.vault.server.ca_cert }}
+      ca_certificate: "{{ .Values.vault.server.ca_cert }}"
+      {{- end }}
+      namespace: "{{ .Values.vault.namespace }}"
+      token:
+        from_disk: "/opt/vault/token"
+  network_address_key_backend:
+    type: "vault"
+    server: "{{ .Values.vault.server.address }}"
+    {{- if .Values.vault.server.ca_cert }}
+    ca_certificate: "{{ .Values.vault.server.ca_cert }}"
+    {{- end }}
+    namespace: "{{ .Values.vault.namespace }}"
+    token:
+      from_disk: "/opt/vault/token"
+
+full_node_networks:
+- network_id:
+    private: "vfn"
+  listen_address: "/ip4/0.0.0.0/tcp/6181"
+  identity:
+    type: "from_config"
+    key: "b0f405a3e75516763c43a2ae1d70423699f34cd68fa9f8c6bb2d67aa87d0af69"
+    peer_id: "00000000000000000000000000000000d58bc7bb154b38039bc9096ce04e1237"
```

Comment:
the format of this peer_id may need to change with recent changes (e.g. https://github.com/aptos-labs/aptos-core/pull/532) see https://github.com/aptos-labs/aptos-core/blob/bb7e5fc7a0b52b5bc634377c2cfc0442ff291b2f/terraform/helm/validator/files/configs/validator.yaml#L83